ntc_templates/templates/fsas_sir_show_ip_route.textfsm

```diff
@@ -0,0 +1,15 @@
+Value FLAG (\S)
+Value PROTOCOL (\w+)
+Value NETWORK (\S+)
+Value PREFIX_LENGTH (\d+)
+Value GATEWAY (\S+)
+Value DISTANCE (\d+)
+Value UPTIME (\S+)
+Value INTERFACE (\w+)
+
+Start
+  ^FP\s+Destination\/Mask
+  ^---\s+
+  ^\s*${FLAG}?${PROTOCOL}\s+${NETWORK}\/${PREFIX_LENGTH}\s+${GATEWAY}\s+${DISTANCE}\s+${UPTIME}\s+${INTERFACE}\s*$$ -> Record
```

Comment:
@caribouHY
You were right, in relation to the `FLAG`, we need either a space or the flag.
Apologies this didn't stand out in my limited testing (I chose to suggest via webui instead of local testing, tisk tisk to myself). :flushed:

```suggestion
  ^(\s|${FLAG})${PROTOCOL}\s+${NETWORK}\/${PREFIX_LENGTH}\s+${GATEWAY}\s+${DISTANCE}\s+${UPTIME}\s+${INTERFACE}\s*$$ -> Record
```